components/checkbox/Checkbox.razor

```diff
@@ -1,9 +1,11 @@
 ﻿@namespace AntDesign
 @inherits AntInputBoolComponentBase
 
-<label class="@ClassMapperLabel.Class" style="@Style" id="@Id" @ref="Ref">
+<label class="@ClassMapperLabel.Class" style="@Style" For="@Id"  @ref="Ref">
```

Comment:
Shouldn't we start with lower case at `For`?

Reply:
It'll render properly with either.  If you'd like for me to make it lowercase I'm happy to.

Reply:
You got it.  Will adjust it in the morning.

Reply:
Updated.